packages/msi/README.md

```diff
@@ -0,0 +1,40 @@
+# MSI Package
+
+## Prerequisites
+
+- [dotnet]
+- [wix]: `dotnet tool install --global wix`
+
+## Build the package
+
+This guide assumes and advices the use of PowerShell as your shell environment for this purpose.
+
+### Set the environment variables
+
+```powershell
+$env:VERSION = "1.3.37"
+$env:INSTALLER= "msi"
+```
+
+### Build the installer
+
+```powershell
+wix build -arch arm64 -out install-arm64.msi
+```
+
+## Install the package
+
+### For the current user
+
+```powershell
+install-arm64.msi MSIINSTALLPERUSER=1 ALLUSERS=2
```

Comment:
`ALLUSERS=2` is what you'd typically author into the `Properties` table. It can be passed via the command line, yes, but it might be worth informing users that "2" is context-sensitive. If they are in an elevated process, the MSI will install per-machine. Upgrades *must* happen in the same context in which the MSI was installed, so if they later try to do a per-user upgrade it'll fail with a rather unintuitive user error message; and the error message buried in a log - none by default - is only slightly more intuitive.

Typically, the default `ALLUSERS` = unset, equivalent to an empty string, will install per-user. If a user wants to do a per-machine install - likely rare given your current acquisition docs - they can pass `ALLUSERS=1`.

See <https://learn.microsoft.com/windows/win32/msi/allusers> for docs, and <https://github.com/Azure/azure-dev/blob/6cd82ae9ff02e156a0056e994215191469442ab3/cli/installer/windows/azd.wxs#L32-L34> for an example I did for the Azure Developer CLI (`azd`).

You should author those into the MSI so that the default install just works as a per-user install, which requires no elevation just like your `brew` installation, etc.

Reply:
I found that adding `Scope="perUserOrMachine"` to the `Package` element does the same. Wouldn't have found out without your comment though!